src/types/box-types.ts

```diff
@@ -1,22 +1,44 @@
-import {ReactNode} from 'react'
+import React from 'react'
 import { EnhancerProps } from './enhancers'
 
-export type BoxProps = EnhancerProps & {
-  /**
-   * Lets you change the underlying element type. You can pass either a
-   * string to change the DOM element type, or a React component type to
-   * inherit another component. The component just needs to accept a
-   * `className` prop to work. A good example is inheriting the react-router
-   * `Link` component.
-   */
-  is?: keyof JSX.IntrinsicElements | React.ComponentClass<any, any> | React.FunctionComponent<any>
+/**
+ * @template T Object
+ * @template K Union of T keys
+ */
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
The latest stable release of TypeScript added an `Omit` type, dunno if we want to use that or rename this type.

Reply:
Ah, good call!